frontend/src/routes/PrivateSection.js

```diff
@@ -17,14 +17,15 @@ const useStyles = createUseStyles({
         minHeight: 850
     },
     mainBlock: {
-        marginLeft: 255,
+        marginLeft: 200,
         padding: 30,
         '@media (max-width: 1080px)': {
             marginLeft: 0
         }
     },
     contentBlock: {
-        marginTop: 54
+        marginTop: 54,
+        marginLeft:0
     }
```

Comment:
This could be responsible for the need of a negative margin in one of the other components ☺️ 📝

Reply:
then we can fix it. And thanks for adding the tab logo item on the browser , it looks cool ! @Waldleufer  ☺️